identity-account/examples/methods.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example methods
+
+use identity_account::account::Account;
+use identity_account::error::Result;
+use identity_account::events::Command;
+use identity_account::identity::IdentitySnapshot;
+use identity_account::storage::MemStore;
+use identity_did::verification::MethodScope;
+use identity_iota::did::IotaDID;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+  pretty_env_logger::init();
+
+  // Create an in-memory storage instance for the account
+  let storage: MemStore = MemStore::new();
+
+  // Create a new Account with the default configuration
+  let account: Account<MemStore> = Account::new(storage).await?;
+
+  // Create a new Identity with default settings
+  let snapshot: IdentitySnapshot = account.create(Default::default()).await?;
+
+  // Retrieve the DID from the newly created Identity state.
+  let document: &IotaDID = snapshot.identity().try_document()?;
```

Comment:
DID or DID Document? The naming suggests DID Document, but the type suggests a DID ;D